MODULE.bazel

```diff
@@ -5,7 +5,7 @@ module(
 )
 
 bazel_dep(name = "bazel_features", version = "1.9.1")
-bazel_dep(name = "bazel_skylib", version = "1.3.0")
+bazel_dep(name = "bazel_skylib", version = "1.6.1")
```

Comment:
Could you please add a note in the `CHANGELOG` that the `bazel_skylib` version has been bumped? I think this detail might be great to see at a glance when we cut the next release.